oam/etc/Columnstore.xml

```diff
@@ -486,10 +487,10 @@
 		<MaxBuckets>128</MaxBuckets>
 		<MaxElems>128K</MaxElems>  <!-- 128 buckets * 128K * 16 = 256 MB -->
 		<FifoSizeLargeSide>64</FifoSizeLargeSide>
-		<PmMaxMemorySmallSide>1G</PmMaxMemorySmallSide>
+		<PmMaxMemorySmallSide>100M</PmMaxMemorySmallSide>
```

Comment:
I disagree with this default. If the cloud needs it then put it into init scripts for the cloud image.

Reply:
A mistake. For testing purposes only. Corrected.